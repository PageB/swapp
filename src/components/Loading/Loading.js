import React from 'react';

import Spinner from './../../assets/images/giphy.gif';
import styles from './Loading.module.scss';

const Loading = props => {
  return (
    <div className={styles.Loading}>
      <img className={styles.Loading__Image} src={Spinner} alt="Loading" />
    </div>
  );
};

export default Loading;
