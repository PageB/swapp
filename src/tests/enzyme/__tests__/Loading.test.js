import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';

import Loading from '../../../components/Loading/Loading';

configure({ adapter: new Adapter() });

describe('Loading Component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Loading />);

    expect(wrapper).toMatchSnapshot();
  });
});
