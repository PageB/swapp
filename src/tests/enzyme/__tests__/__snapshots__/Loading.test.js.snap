// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Component should match snapshot 1`] = `
<div
  className="Loading"
>
  <img
    alt="Loading"
    className="Loading__Image"
    src="giphy.gif"
  />
</div>
`;
